### DISCUSSION TITLE
Influence of Elmish.Uno?

### DISCUSSION THREAD

**Discussion by houstonhaynes** [JordanMarr/ReactiveElmish.Avalonia#21] [General]
I'm just curious whether you've taken any notes from Elmish.Uno while working on Elmish.Avalonia. My friend [Andrii Chebukin](https://github.com/xperiandri) shared some of his notes on working through (and in some cases enhancing) Elmish.Uno and it happens to dovetail pretty well with some of the ideas I'm flirting with right now. So this question is more about me "zooming out" to see what's happening in the Elmish world.

**Comment by JordanMarr** [comment]
TBH, I didn't even know that project existed, but it looks like he also branched off of the Elmish.WPF project.
One thing I love about Elmish in F# is that it is almost universally embraced by all F# UI libraries, so your investment in learning Elmish carries _a lot_ of forward momentum! 
This has not been my experience in any other ecosystem.

**Comment by houstonhaynes** [comment]
Agreed. It's funny that they specifically mention something I'm already considering - which is how to segment the project in a way that I can build a Fable/React version for browser from the same code base and avoid WASM (for now).
